common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -54,6 +54,7 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
     private V[] values;
     private Collection<V> valueCollection;
```

Comment:
Any thoughts on if this can be dropped?  I wonder if this may provide unexpected concurrent modification type exceptions by caching this.